File: src/main/java/org/taktik/icure/services/external/rest/v1/dto/PaymentDto.java
package org.taktik.icure.services.external.rest.v1.dto;

import org.taktik.icure.entities.embed.PaymentType;

public class PaymentDto {

    private long paymentDate;
    private PaymentType paymentType;
    private Double paid;

    public long getPaymentDate() {
        return paymentDate;
    }

    public void setPaymentDate(long paymentDate) {
        this.paymentDate = paymentDate;
    }

    public PaymentType getPaymentType() {
        return paymentType;
    }

    public void setPaymentType(PaymentType paymentType) {
        this.paymentType = paymentType;
    }

    public Double getPaid() {
        return paid;
    }

    public void setPaid(Double paid) {
        this.paid = paid;
    }
}

File: src/main/kotlin/org/taktik/icure/services/external/rest/v1/dto/embed/PaymentDto.kt
package org.taktik.icure.services.external.rest.v1.dto.embed

import com.github.pozo.KotlinBuilder
import java.io.Serializable

@KotlinBuilder
data class PaymentDto(
        val paymentDate: Long = 0,
        val paymentType: PaymentTypeDto? = null,
        val paid: Double? = null
) : Serializable
